### PR TITLE
feat: Add periods to patient addresses

### LINKED ISSUE #2913: Add effective dates to Patient address
## Description

We need to add effective dates to addresses so it is clear what is a current address and what is a past address 

## Steps to reproduce
Example ECR
1. TN 6 

## Expected Result

We would like to display all information that is available to (consult w/ design first).

- [ ] If there are multiple addresses, only show the current address in the patient summary 
- [ ] If the are multiple addresses, show them all in Patient Info with effective dates 

## Actual result

- [ ] There are no dates and current address is unknown 

## Priority

_Provide an estimate for level of impact the bug has today by placing an X in the applicable bracket_

#### Priority
- [ ] **Low priority** _The bug does not have to be fixed immediately. High and medium priority bugs should be addressed first._
- [X] **Medium priority** _Typically, medium-priority bugs do not affect customers directly and therefore can be fixed in the normal course of testing and development._
- [ ] **High priority** _High priority bugs must be addressed immediately. They often affect the app’s functionality and impact customers and their user experience. They must take priority._

## Notes

Effective dates for address will need to be added to the fhir converter

### PR DESCRIPTION
**Pull request by mcmcgrath13**
# PULL REQUEST

## Summary
Add the useable period to a patient's addresses if available. Counterpart of https://github.com/skylight-hq/FHIR-Converter/pull/20

Refactored `formatAddress` to be a bit more flexible and terser to use on the common case in the process.

## Related Issue
Fixes #2913

## Acceptance Criteria
We would like to display all information that is available to (consult w/ design first).

- [x] If there are multiple addresses, only show the current address in the patient summary 
- [x] If the are multiple addresses, show them all in Patient Info with effective dates 

## Additional Information
image

image

### CODE REVIEW

File: containers/ecr-viewer/seed-scripts/create-seed-data.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def _process_files(args):
     :return: A list of fhir bundles
     """
     print("Converting files...")
-    subfolders = ["LA"]
+    subfolders = ["TN"]
```
Comment (mcmcgrath13) [inline containers/ecr-viewer/seed-scripts/create-seed-data.py:33]:
```suggestion
    subfolders = ["LA"]
```

File: containers/fhir-converter/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
 
 # Download FHIR-Converter
-RUN git clone https://github.com/skylight-hq/FHIR-Converter.git --branch v7.0-skylight-15 --single-branch /build/FHIR-Converter
+RUN git clone https://github.com/skylight-hq/FHIR-Converter.git --branch mcm/econ-periods --single-branch /build/FHIR-Converter
```
Comment (mcmcgrath13) [inline containers/fhir-converter/Dockerfile:4]: todo: update this with released version once merged

**Review by JNygaard-Skylight — APPROVED** [review]
LGTM!

File: containers/fhir-converter/Dockerfile (hunk 2 above)
Comment (mcmcgrath13) [inline containers/fhir-converter/Dockerfile:4]:
```suggestion
RUN git clone https://github.com/skylight-hq/FHIR-Converter.git --branch v7.0-skylight-16 --single-branch /build/FHIR-Converter
```

**Pull request merged**